### ISSUE TITLE
QR Code Entry Check-in says my password is wrong

### ISSUE THREAD

**Issue by etheisen** [geoffhumphrey/brewcompetitiononlineentry#1081]
Description of Issue: QR Code Entry check-in is available, but it keeps telling me my password is wrong. I have reset my password and retried logging in, but it keeps telling me my password is wrong. Is there any documentation on how this function works so I can troubleshoot on my own?

https://bellinghamhomebrewersguild.org/bestofthebay/#

**Comment by geoffhumphrey** [comment]
@etheisen - apologies for the late reply. Unfortunately, there really isn't any documentation. It's a pretty simple function of checking a password against a stored hash in the DB. Were you able to resolve your issue?

**Comment by etheisen** [comment]
No, I wasn't able to get it to work at all. I even changed my password and
tried to get it to work. I assume this uses the user password, not like a
super admin password. I really would like to try to get this working for
next year. Can someone put together the most basic of documentation on how
it works? These are the steps I went through:

   1. I downloaded a QR code reader (I have an android, Moto z3 play)
   2. I then scanned a QR code, it takes me to the qr.php page with the ID
   number in the query string
   3. I put in my password
   4. It tells me my password is incorrect.
   5. I login to the system on the front end, update my password.
   6. I then loop back to step 1, and take the updated password (which I
   copied) and paste it in.
   7. It tells me my password is incorrect

What am I doing wrong here? That's why I am wondering is there a super
admin password, or do I not have a proper permission in the system. I am
set up as the Cellar Master.

**Comment by etheisen** [comment]
Any idea why this is not working? and pretty much has never worked for me?

**Comment by etheisen** [comment]
I am really hoping to get this solved before next August, can anyone help me with this? I've got no idea why this isn't working for me. 

Are there device requirements? Is it a login issue? I"ve tried on both my iPad and my Android phone, so I don't know what else to do.

**Comment by etheisen** [comment]
There must be some way to test why this isn't working?

**Comment by grmelacz** [comment]
I've just tested the same problem as you have as I've forgotten our check-in password. Updated the password in the preferences, then I was able to successfully scan a code and log in with the entered password. I would suggest you updating to the last BCOE&M version.

**Comment by etheisen** [comment]
Where do you set the check-in password?

**Comment by grmelacz** [comment]
Admin > Competition Preparation > Competition Info > Edit > QR Code Log On Password > Add, Update, ...

<img width="595" alt="Screenshot 2021-07-31 at 13 16 01" src="https://user-images.githubusercontent.com/41264018/127738179-afabeb0a-ec72-4b3a-8d1b-461f4b2c05c4.png">

**Comment by etheisen** [comment]
Wow, this would have been good to have a couple weeks ago. So, is this only accessible to a superadmin? I'm the Cellar master and I don't have access to reset the QR code... Our webmaster deals with these things... he's not the the cellar master.
